packages/flutter/lib/src/rendering/flex.dart

```diff
@@ -117,7 +117,7 @@ class FlexParentData extends ContainerBoxParentData<RenderBox> {
   /// non-zero, the amount of space the child's can occupy in the main axis is
   /// determined by dividing the free space (after placing the inflexible
   /// children) according to the flex factors of the flexible children.
-  int? flex;
+  num? flex;
```

Comment:
Can you add a note to the docs about why this is typed `num` and that you'd typically pass in an `int` (or `double`)?

Reply:
Sure, I updated it in [doc: Update the doc](https://github.com/flutter/flutter/pull/147976/commits/593490d596a6bfb006e657be5915606358d147ec)